_posts/2016-02-24-session-user-flow.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Use session variables to optimize your user flow"
+date: 2016-02-24 13:37:00
+categories: ["ruby", "rails", "sessions"]
+author: "schmierkov"
+---
+
+Sessions provide you a nice little data storage feature where the application does not need to get the informations directly from the database. So you do not have to persist data in your database and can easily store infos to a user on the fly. This is a nice way to enhance the user experience on your page. Let's say you want to show all users, from a specific area, a new fancy sign up form and the rest only sees the old form. With storing these location informations in a session variable, you do not need to persist this infos in your database.
```

Comment:
"can easily store infos to a user on the fly" should be "can easily store info for a user on the fly" (information doesn't have a plural) 

I'd split up this big paragraph into two. For example: 

"This is a nice way to enhance the user experience on your page. </p><p> Let's say you want to"

This "Let's say you want to show all users, from a specific area, a new fancy sign up form and the rest only sees the old form" could sound better like this: "Let's say that you want to show some users a new fancy sign up form and the rest the old form."